P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -1626,13 +1626,14 @@ editorial practices:
   <specDesc key="unclear" atts="reason"/>
   <specDesc key="add"/>
   <specDesc key="del"/>
+  <specDesc key="ellipsis"/>
 </specList></p>
 <p>Encoders may choose to omit parts of the copy text for reasons
 ranging from illegibility of the source or impossibility of transcribing
 it, to editorial policy, e.g. a systematic exclusion of poetry or prose
 from an encoding. The full details of the policy decisions concerned
 should be documented in the TEI header (see section <ptr target="#HD5"/>).
-Each place in the text at which omission has taken place should be
+When text is illegible or impossible to decipher, this should be
```

Comment:
I don't like this rewrite, which changes the meaning significantly. It would be good to discuss here or later the relationship between `<gap>` and `<ellipsis>`. My understanding is that gap is for unmarked omissions of text or for illegible text while ellipsis is for marked, omitted text.

Reply:
Put another way, `<gap>` *must* be a legal way to mark omitted text (as well as illegible text).

Reply:
@hcayless I think the only way to clarify this is to expand the explanation here to include both `<gap>` and `<ellipsis>`, so the rewrite will need to be substantial, for the whole paragraph. If we discuss these together, we should probably rearrange the section so that the detailed explanation and example of `<ellipsis>` follows immediately after `<gap>`.

I'm going to try rewriting the paragraph here. What do you think of this? New material is marked in bold: 

Encoders may choose to omit parts of the copy text for reasons ranging from illegibility of the source or impossibility of transcribing it, to editorial policy, e.g. a systematic exclusion of poetry or prose from an encoding. The full details of the policy decisions concerned should be documented in the TEI header (see section 2.3 The Encoding Description). 
**When omissions are not deliberately signaled in the source document,** each place in the text at which omission has taken place should be marked with a `<gi>gap</gi>` element, with optionally further information about the reason for the omission, its extent, and the person or agency responsible for it, as in the following examples:

[. . . examples of `<gap>` follow . . .]

Distinct from `<gi>gap</gi>`, the `<gi>ellipsis</gi>` element provides a means to encode the deliberate, marked omission of content in a source document. Within `<gi>ellipsis</gi>`, `<gi>metamark</gi>` describes the denotation of omitted material in the source, and encoders may optionally provide a `<gi>note</gi>` describing the omission and `<gi>supplied</gi>` to indicate the content omitted at this point.

Reply:
I've been tinkering with this rewriting, so please read on GitHub.